src/Symfony/Component/AssetMapper/AssetMapperRepository.php

```diff
@@ -34,6 +34,7 @@ public function __construct(
         private readonly string $projectRootDir,
         private readonly array $excludedPathPatterns = [],
         private readonly bool $excludeDotFiles = true,
+        private readonly bool $strictMode = false
```

Comment:
or just $debug? "strict" sounds like a promise with no definition to me :)

Reply:
The problem here is there is some counter-intuitive logic.... Beeing strict when in dev and allowing to bypass some checks in PROD feels a bit weird.

And i don't think we should set "not strict" per default in prod.

Reply:
I agree that `$debug` is a better choice. That has been changed!